lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/KbvQuality.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.di.ipv.cri.kbv.api.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public enum KbvQuality {
+    LOW(1),
+    MEDIUM(2),
+    HIGH(3);
+
```

Comment:
Currently, `MEDIUM` and `HIGH` are not being referenced.

Reply:
It doesn't matter, as it would be used in the next PR